src/main/java/com/yugabyte/sample/apps/AppConfig.java

```diff
@@ -125,8 +127,8 @@
   // Name of the default username for postgres.
   public String defaultPostgresUsername = "postgres";
 
-  // Does the table need to be dropped.
-  public boolean shouldDropTable = false;
+  // should we drop / truncate table
```

Comment:
Comments begin with upper case, and end with period (as per our coding convention).